### PR TITLE
HdmiCec_2 json and md files updated as per code

### PR DESCRIPTION
**Pull request by pjayar213**
Documentation enhancement for HdmiCec_2 plugin
Jira ticket link: https://ccp.sys.comcast.net/browse/RDK-35404

### CODE REVIEW

File: HdmiCec_2/HDMICec_2.json [hunk 1]
```diff
@@ -45,8 +60,52 @@
...
+                "required": [
+                    "numberofdevices",
+                    "deviceList",
+                    "success"
+                ]
+            }
+        },
         "getEnabled": {
-            "summary": "Returns whether HDMI-CEC is enabled",
+            "summary": "Returns whether HDMI-CEC is enabled.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline HdmiCec_2/HDMICec_2.json:108]: Returns whether HDMI-CEC driver enabled status

File: HdmiCec_2/HDMICec_2.json [hunk 2]
```diff
@@ -64,7 +123,7 @@
             }
         },
         "getOSDName":{
-            "summary": "Returns the OSD name used by device",
+            "summary": "Returns the OSD name used by device.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline HdmiCec_2/HDMICec_2.json:126]: name set by the user

File: HdmiCec_2/HDMICec_2.json [hunk 3]
```diff
@@ -82,7 +141,7 @@
             }
         },
         "getOTPEnabled":{
-            "summary": "Returns whether HDMI-CEC OPT option is enabled",
+            "summary": "Returns whether HDMI-CEC OTP option is enabled.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline HdmiCec_2/HDMICec_2.json:144]: DMI-CEC OTP option enable status

File: HdmiCec_2/HDMICec_2.json [hunk 4]
```diff
@@ -191,5 +256,63 @@
...
+                        "$ref": "#/definitions/logicalAddress"
+                    }
+                },
+                "required": [
+                    "logicalAddress"
+                ]
+            }
+        },
+        "onDeviceInfoUpdated": {
+            "summary": "Triggered when device information changes (vendorID, osdName)",
```
Comment (apatel859) [inline HdmiCec_2/HDMICec_2.json:276]: Triggered when device system information updated

File: HdmiCec_2/HDMICec_2.json (hunk 1 above)
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

File: HdmiCec_2/HDMICec_2.json (hunk 2 above)
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

File: HdmiCec_2/HDMICec_2.json [hunk 5]
```diff
@@ -100,7 +159,7 @@
             }
         },
         "getVendorId":{
-            "summary": "Returns the vendor ID used by the device",
+            "summary": "Returns the vendor ID used by the device.\n  \n### Events \n\n No Events",
```
[the remark this answers is not in the document]
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

File: HdmiCec_2/HDMICec_2.json [hunk 6]
```diff
@@ -141,7 +206,7 @@
             }
         },
         "setOSDName":{
-            "summary": "Sets the OSD name",
+            "summary": "Sets the OSD name used by the device.\n  \n### Events \n\n No Events",
```
[the remark this answers is not in the document]
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

File: HdmiCec_2/HDMICec_2.json [hunk 7]
```diff
@@ -175,7 +240,7 @@
             }
         },
         "setVendorId":{
-            "summary": "Sets the vendor ID used by the device",
+            "summary": "Sets the vendor ID used by the device.\n  \n### Events \n\n No Events",
```
[the remark this answers is not in the document]
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

File: HdmiCec_2/HDMICec_2.json (hunk 4 above)
Comment (pjayar213) [reply HdmiCec_2/HDMICec_2.json]: Updated changes as per your review comments

**Review by apatel859 — APPROVED** [review]

**Review by anand-ky — APPROVED** [review]

**Pull request merged**